config/environments/production.rb

```diff
@@ -64,6 +65,12 @@
   # config.active_job.queue_name_prefix = "bestcode_#{Rails.env}"
 
   config.action_mailer.perform_caching = false
+  config.action_mailer.default_url_options = { host: "bestcode.reviews", protocol: "https" }
+  config.action_mailer.delivery_method = :mailgun
+  config.action_mailer.mailgun_settings = {
+    api_key: ENV["MAILGUN_API_KEY"],
+    domain: "mg.bestcode.reviews"
```

Comment:
__[RuboCop]__

Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.


<!-- Comment by [Sider](https://sider.review) -->